cmd/generator/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"os"
+)
+
+func main() {
+	argsWithProg := os.Args
+
+	if len(argsWithProg) != 2 {
+		panic("Wrong amount of arguments!")
```

Comment:
panic here is also not good, because it will print stacktrace, which is useless for CLI tool

Reply:
Ok, I would use log.Fatalln("Wrong amount of arguments!")